### PR TITLE
feat: Support for DID Factom

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: cmd/vc-rest/startcmd/start.go [hunk 1]
```diff
@@ -501,7 +502,7 @@ func supportedMode(mode string) bool {
 // acceptsDID returns if given did method is accepted by VC REST api
 func acceptsDID(method string) bool {
 	return method == didMethodVeres || method == didMethodElement || method == didMethodSov ||
-		method == didMethodWeb || method == didMethodKey
+		method == didMethodWeb || method == didMethodKey || method == didMethodFactom
```
Comment (troyronda) [inline cmd/vc-rest/startcmd/start.go:505]: I'm somewhat curious why we don't just reply true :)

Comment (troyronda) [reply cmd/vc-rest/startcmd/start.go:505]: do we have multiple DID acceptors in this service?
if we need to do it this way, we need a followup to make this configurable.

Comment (rolsonquadras) [reply cmd/vc-rest/startcmd/start.go:505]: May be original idea was to have some whitelist. Probably we can move this to a config.

Comment (rolsonquadras) [reply cmd/vc-rest/startcmd/start.go:505]: https://github.com/trustbloc/edge-service/issues/406

**Pull request merged**